cowamm/cow_amm_cow_with_users_surplus_query_4031724.sql

```diff
@@ -1,12 +1,11 @@
 -- This query computes how much surplus has been provided to CoW AMMs, when trading with other user orders
```

Comment:
Can we move this file into a bounty subfolder to keep the main cow amm folder lean?